manage-data/data-store/data-streams/query-downsampled-data.md

```diff
@@ -2,7 +2,7 @@
 applies_to:
   stack: ga
   serverless: ga
-navigation_title: "Query downsampled data"
+navigation_title: "Querying"
```

Comment:
Can't suggest a change on the H1 itself, but maybe SEO-wise it should be `# Querying downsampled data in {{es}}`